Mailer/app/api/v1/Handlers/Mailers/MailLogsCountHandler.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Remp\MailerModule\Api\v1\Handlers\Mailers;
+
+use Nette\Utils\Json;
+use Nette\Utils\JsonException;
+use Remp\MailerModule\Repository\ListsRepository;
+use Remp\MailerModule\Repository\LogsRepository;
+use Remp\MailerModule\Repository\MailTypesRepository;
+use Tomaj\NetteApi\Handlers\BaseHandler;
+use Tomaj\NetteApi\Params\InputParam;
+use Tomaj\NetteApi\Response\JsonApiResponse;
+use Tracy\Debugger;
+
+class MailLogsCountHandler extends BaseHandler
+{
+    private $logsRepository;
+
+    public function __construct(
+        LogsRepository $logsRepository
+    ) {
+        parent::__construct();
+        $this->logsRepository = $logsRepository;
+    }
+
+    public function params()
+    {
+        return [
+            new InputParam(InputParam::TYPE_POST, 'where', InputParam::OPTIONAL),
+        ];
+    }
+
+    public function handle($params)
+    {
+        $output = $this->logsRepository->all();
+
+        if (!empty($params['where'])){
```

Comment:
Never take external conditions and use them directly in query. Always require values only for parameters you want to allow. See for example [this hander](https://github.com/remp2020/crm-users-module/blob/master/src/api/UserMetaKeyUsersHandler.php) and its [schema](https://github.com/remp2020/crm-users-module/blob/master/src/api/user-meta-key-users.schema.json).

_(that's why we want JSON payload and validation of schema)_

Reply:
This implementation is highly prone to errors as it expects caller is aware we're using Nette Database. This shouldn't be a concern for API caller.